src/theme/variables.scss

```diff
@@ -118,3 +117,26 @@ $colors: (
 	.alignright {
 		text-align: right !important;
 	}
+	.alignleft {
+		text-align: left !important;
```

Comment:
I wonder if ionic has built in alignment CSS stuffs. Did you look for that at all?